ReactAndroid/src/main/java/com/facebook/react/modules/systeminfo/AndroidInfoModule.java

```diff
@@ -27,6 +29,19 @@
 
   private static final String IS_TESTING = "IS_TESTING";
 
+  private @Nullable ReactApplicationContext mReactApplicationContext;
+  private boolean isTablet;
+
+  public AndroidInfoModule(ReactApplicationContext reactContext) {
+    this((Context) reactContext);
+    mReactApplicationContext = reactContext;
+  }
+
+  public AndroidInfoModule(Context context) {
+    mReactApplicationContext = null;
+    isTablet = context.getResources().getConfiguration().smallestScreenWidthDp >= 600;
+  }
```

Comment:
If you need the `reactContext` you should change this module to extend `ReactContextBaseJavaModule`